.github/workflows/first-time-contributor.yml

```diff
@@ -8,32 +8,33 @@ jobs:
     if: >-
       (github.repository == 'meshery/meshery') &&
```

Comment:
are `( <expression> )` is valid syntax or you mean `${{ <expression> }}` as mentioned in the docs https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#about-expressions

Reply:
@aryannargotra are you incorporating this feedback?